docs/userguide.rst

```diff
@@ -221,3 +221,31 @@ were deployed during ``init``, you can run:
 .. code-block:: console
 
    $ reana-cluster down
+
+Delete interactive sessions
+---------------------------
+
+Interactive sessions that were not closed after work have been finished leave
+active Kubernetes objects (pod, service and ingress) running in the Reana cluster.
+These objects might need to be deleted from time to time. Please note that
+interactive session pod and service are linked to the ingress object. If ingress
+object gets deleted other twos will be deleted automatically.
+
+To delete open interactive sessions we suggest to use following commands:
+
+.. code-block:: console
+
+   # get all ingresses of open interactive sessions
+   $ kubectl get ingresses | grep interactive
+   interactive-jupyter-ad814137-baec-4ee5-b899-1d549e90b44a   *                 80      103m
+   interactive-jupyter-e4c7aa38-370b-47b7-bf8c-44f5f0f44b44   *                 80      3h48m
+   interactive-jupyter-ec5438b1-65a8-4bba-a52c-ff7970242e83   *                 80      3h16m
+   interactive-jupyter-f401874d-e9a6-4b2b-9b9d-12b0f141a442   *                 80      103m
+   interactive-jupyter-f9f56f92-a533-4bcc-aad9-581f5e31ff0f   *                 80      103m
+   # lets delete all remaining objects of open interactive sessions
+   $ kubectl delete ingresses $(kubectl get ingresses | grep interactive | awk '{print $1}')
```

Comment:
I think we can save the `awk` call by doing `kubectl get ingress -o name | grep interactive`.

Reply:
It would be great if we could set an `age` to filter the sessions but I don't see an easy way now..

Reply:
It won't work.

```
$ kubectl delete pods $(kubectl get pods -o name)
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
```

I don't have any ingresses running now, so example is with pods.

```
 $ kubectl get pods -o name
pod/db-6d845f7ff9-wsxl8
pod/job-controller-6d7c6bd659-r7wzw
pod/message-broker-5bf98cc4f9-vzlv6
pod/server-66f4d75c47-mg9wc
pod/wdb-6b794c6855-j52sq
pod/workflow-controller-b6cbf4cf7-59x5k
```

I don't find a way to get rid of ``pod/``. Maybe I am missing something.